docs/activities.md

```diff
@@ -0,0 +1,96 @@
+# Sprint Activities List
+
+## Most popular
+
+### Wine tasting - 35€
+
+Dive deeper into the unique style of how wine is produced on Lanzarote.
+
+Choose four of your favourite styles from a portfolio of wines.
+
+A bite of artisanal cheese and charcuterie will accompany your personalized tasting. The experience takes about 60 minutes.
+
+### Vulcano hike - free
+
+A classic of Ocean Sprint. We leave the venue for a 1.5h hike to the nearest vulcano with great views over the nearby tows and coastline.
+
+Beginner-friendly.
+
+### Intro to surfing - 30€
+
+Pickup by vans at the sprint venue and leave for Famara, the surfing mecca of Lanzarote. Two hours in the water with rented surfboard, wetsuits and surf coaching. Back by dinner.
+
+Beginner-friendly.
+
+### Bonfire Night in the Desert - free
+
+A night under the stars with a bonfire in the middle of a vulcano. 
+
+We'll organize the transportation and each participants brings their own food&drinks.
```

Comment:
How does "each participants brings their own food&drinks" work? They go to dino before we leave? 

Is this instead of dinner one day? What do the rest do?

Reply:
yeah, we'd get folks to dino.

I'd recommend we have dinner beforehand somewhere because it's annoying to eat there with 30 people and no tables.

Reply:
Write that in the description. Dinner first, then off to dino for extras and drinks.